app/src/main/java/com/github/se/polyfit/model/meal/MealTag.kt

```diff
@@ -0,0 +1,62 @@
+package com.github.se.polyfit.model.meal
+
+import android.util.Log
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+
+data class MealTag(var tagName: String, var tagColor: Color) {
```

Comment:
It might be better to use an enum instead of a String for the tagName. This would ensure uniformity and potentially prevent errors related to string handling.

Reply:
The reason why I used String for the tagName is for allowing user to create their own custom tags, which could be anything. Do you think we could keep string for tagname or just don't allow users to create tags?

Reply:
I will update the local database but also isnt the firebase database just the serialization and deserialization?

Reply:
Yes, just both of those functions need to be updated.

Reply:
i think I already did it for both MealTag and Meal, am I missing anywhere else for the serialization?

Reply:
No that is all. Everything looks good !